### DISCUSSION TITLE
Error when running monailable with bundle lung nodule detection

### DISCUSSION THREAD

**Discussion by anuragp2018** [Project-MONAI/MONAILabel#1635] [General]
when i try to run monailable server using this command monailabel start_server --app apps/radiology  --studies D:\Task06_Lung\Task06_Lung\imagesTr --conf models segmentation --conf bundles lung_nodule_ct_detection_v0.2.0

it start the server but when i run auto segmentation in slicer it show error internal server not found.

**Comment by diazandr3s** [comment]
Hi @anuragp2018,

Are you running both the 3DSlicer and MONAI Label server on the same machine? 
Can you please provide more information about this?

Let us know

**Reply by anuragp2018** [reply]
yes
